check_for_palindromes.js

```diff
@@ -0,0 +1,16 @@
+function palindrome(str) {
+  
+  /* remove special characters, spaces and make lowercase*/
+  var removeChar = str.replace(/[^A-Z0-9]/ig, "").toLowerCase();
```

Comment:
skoro nie tylko usuwasz znaki, ale rowniez normalizujesz, to lepiej byloby nazwac to normalizedString.